user-manager/src/main/java/com/qpay/usermanager/model/entity/merchant/MerchantEntity.java

```diff
@@ -0,0 +1,42 @@
+package com.qpay.usermanager.model.entity.merchant;
+
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import jakarta.persistence.Entity;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+import java.time.LocalDateTime;
+
+@Data
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@AllArgsConstructor(access = AccessLevel.PROTECTED)
```

Comment:
why u use AccessLevel.PROTECTED?

Reply:
as roma said, 

> it would be good if constructors have scope only in package or in inheritance classes, because creating instances of this class have to be done by @builder